molo/polls/tests/test_voting.py

```diff
@@ -50,6 +52,46 @@ def test_voting_once_only(self):
             kwargs={'poll_id': question.id}))
         self.assertContains(response, '100%')
 
+    def test_multiple_options_with_translations(self):
+        '''
+        Test that voting does not return an error some an not all
+        choices are translated
+        '''
+        self.client.login(
+            username=self.superuser_name,
+            password=self.superuser_password
+        )
+        setting = SiteSettings.objects.create(site=self.main.get_site())
+
+        setting.show_only_translated_pages = True
+        setting.save()
+        question = Question(title='is this a test', language=self.english,
+                            allow_multiple_choice=True, show_results=False)
+        self.polls_index.add_child(instance=question)
+        # translate the question
+        self.client.post(reverse(
+            'add_translation', args=[question.id, 'fr']))
+        question.save_revision().publish()
+        choice1 = self.make_choice(title='yes',
+                                   parent=question, language=self.english)
+        choice2 = self.make_choice(title='no', parent=question,
+                                   language=self.english)
+        client = Client()
+        client.login(username='superuser', password='pass')
```

Comment:
I don't think you need to re-login because you already have a client with a logged in super user.